message_chunker.py

```diff
@@ -0,0 +1,33 @@
+import nltk
+
+def message_chunk(command):
+    words = nltk.word_tokenize(command)
```

Comment:
Please add a docstring explaining what this method does! (See Multi-line docstrings on [here](https://www.python.org/dev/peps/pep-0257) as a reference)